modules/minimongo-observable/observable-cursor.ts

```diff
@@ -79,13 +80,31 @@ export class ObservableCursor<T> extends Observable<T[]> {
     });
   }
 
+  addedAt(doc, at, before) {
+    this.data.splice(at, 0, doc);
+    this.handleChange();
+  }
+
+  changedAt(doc, old, at) {
+    this.data[at] = doc;
```

Comment:
Ok, I looked a bit closely. Actually you are right it iterates over whole collection which accumulates all subscription on the client. Not sure how #5678 is related to the fetch, anyways it seems to have been addressed. So if you correct to emit this.data and also rename new methods and this.data itself to start with the underscore, I'll merge this PR.